### PR TITLE
Release v2.8.0

### PR DESCRIPTION
**Pull request by zhiqiangjin**
### Notes:
- Supported Controller version: **UserConnect-5.1.1179** and **UserConnect-5.2.1956** 
- Supported Terraform version: **v0.12.x**

### Features:
1. Added support for aws gov access account in **aviatrix_account**
2. Added support for ``customized_routes`` and ``disable_local_route_propagation`` in **aviatrix_aws_tgw_vpc_attachment**
3. Added a link to view the feature compatibility doc online
4. New resources:
  - **aviatrix_aws_tgw_directconnect**

### Enhancements:
1. Added support for editing in **aviatrix_site2cloud**
2. Added blocking setting ``back_gateway_name`` as non-ha_gateway of primary gateway in **aviatrix_site2cloud**

### Bug Fixes:
1. Fixed the issue where it is unable to manage vpn gw w/split tunnel attributes after creation in **aviatrix_gateway**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
-## 2.7.1 (Unreleased)
+## 2.8.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.1.1179** and **UserConnect-5.2.1956**
```
Comment (arvindsree) [inline CHANGELOG.md]: Do we support both?

**Review by arvindsree — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zhiqiangjin) [reply CHANGELOG.md]: Yes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
-## 2.7.1 (Unreleased)
+## 2.8.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.1.1179** and **UserConnect-5.2.1956** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Added support for aws gov access account in **aviatrix_account**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: Added support for AWS GovCloud in **aviatrix_account**

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+2. Added support for ``customized_routes`` and ``disable_local_route_propagation`` in **aviatrix_aws_tgw_vpc_attachment**
+3. Added a link to view the feature compatibility doc online
+4. New resources:
+  - **aviatrix_aws_tgw_directconnect**
+
+### Enhancements:
+1. Added support for editing in **aviatrix_site2cloud**
+2. Added blocking setting ``back_gateway_name`` as non-ha_gateway of primary gateway in **aviatrix_site2cloud**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: 1. Added support for updating in **aviatrix_site2cloud** by ForceNew
2. Blocked setting a non-backup peering-HA gateway as the ``backup_gateway_name`` in **aviatrix_site2cloud**

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+
+### Bug Fixes:
+1. Fixed the issue where it is unable to manage vpn gw w/split tunnel attributes after creation in **aviatrix_gateway**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: Fixed an issue that caused an inability to manage a VPN gateway's ``split_tunnel`` attributes after creating the **aviatrix_gateway**

**Pull request merged**